Source/Sentry/Sentry_Android_UPL.xml

```diff
@@ -37,10 +43,27 @@
         <insert>
             -dontwarn com.sentry.unreal.**
             -keep class com.sentry.unreal.** { *; }
+            -keep class io.sentry.** { *; }
             -keep interface com.sentry.unreal.** { *; }
+            -keep interface io.sentry.** { *; }
         </insert>
     </proguardAdditions>
 
+    <buildGradleAdditions>
+        <if condition="bUploadSymbols">
+            <true>
+                <insert>
+                    apply plugin: 'io.sentry.android.gradle'
+
+                    sentry {
+                        uploadNativeSymbols = true
+                        includeNativeSources = true
```

Comment:
I might be reading this wrong but does that mean it's enabled by default?

Reply:
This entry will end up in a resulting `build.gradle` only if user enabled automatic debug symbols upload in plugin settings (see the `bUploadSymbols` check right above)

Reply:
Ah, gotcha, so this is just the template. Thanks!